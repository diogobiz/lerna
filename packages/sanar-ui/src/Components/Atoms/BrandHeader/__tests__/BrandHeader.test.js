import React from 'react'
import TestRenderer from 'react-test-renderer'

import ESBrandHeader from '..'

it('renders correctly', () => {
    const component = (
        <ESBrandHeader>
            <h1>ESBrandHeader</h1>
        </ESBrandHeader>
    )

    const tree = TestRenderer.create(component).toJSON()
    expect(tree).toMatchSnapshot()
})
