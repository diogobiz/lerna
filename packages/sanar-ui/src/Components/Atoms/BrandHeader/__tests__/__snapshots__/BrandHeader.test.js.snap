// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="es-brand-header"
>
  <img
    alt="Sanar Logo in brand header."
    src="{}"
  />
</div>
`;
